learn/cookbooks/laravel-scout.mdx

```diff
@@ -0,0 +1,188 @@
+description: Learn how to use Meilisearch with Laravel Scout.
+sidebarDepth: 3
+---
+
+# Laravel Scout guide
+
+In this guide, you will learn to setup [Laravel Scout](https://laravel.com/docs/10.x/scout) to use Meilisearch in your Laravel 10 application.
+
+## Prerequisites
+
+Before you start, make sure you have the following installed on your machin:
+- PHP
+- [Composer](https://getcomposer.org/)
+
+You will also need a Laravel application. If you don't have one, you can create a new one by running the following command:
+
+```sh
+composer create-project laravel/laravel my-application
+```
+
+## Installing Laravel Scout
+
+Laravel comes with out-of-the-box full-text search capabilities via Laravel Scout.
+
+To enable it, navigate to your Laravel application directory and install Scout via the Composer package manager:
+
+```sh
+composer require laravel/scout
+```
+
+After installing Scout, you need to publish the Scout configuration file. You can do this by running the following artisan command:
+
+```sh
+php artisan vendor:publish --provider="Laravel\Scout\ScoutServiceProvider"
+```
+
+This command should create a new configuration file in your application directory: `config/scout.php`.
+
+## Configuring the Laravel Scout driver
+
+Let’s configure Laravel Scout to use the Meilisearch driver. First, install the dependencies required to use Scout with Meilisearch via Composer:
+
+```sh
+composer require meilisearch/meilisearch-php http-interop/http-factory-guzzle
+```
+
+Then, update the environment variables in your `.env` file:
+
+```sh
+SCOUT_DRIVER=meilisearch
+# Use the host below if you're running Meilisearch via Laravel Sail
+MEILISEARCH_HOST=http://meilisearch:7700
+MEILISEARCH_KEY=masterKey
+```
+
+### Local development
+
+Laravel’s official Docker development environment, Laravel Sail, comes with a Meilisearch service out-of-the-box. Please note that when running Meilisearch via Sail, Meilisearch’s host is `http://meilisearch:7700`.
+
+<Capsule intent="note">
+This comes from Docker [Bridge network driver](https://docs.docker.com/network/drivers/bridge/#differences-between-user-defined-bridges-and-the-default-bridge) behavior.
+</Capsule>
+
+### Running in production
+
+For production use cases, we recommend using a managed Meilisearch via [Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=laravel&utm_source=docs&utm_medium=laravel-scout-guide). On Meilisearch Cloud, you can find your host URL in your project settings.
+
+<Capsule intent="tip">
+Read our [Meilisearch Cloud quick start](/learn/getting_started/cloud_quick_start?utm_campaign=laravel&utm_source=docs&utm_medium=laravel-scout-guide).
+</Capsule>
+
+## Making Eloquent models searchable
+
+With Scout installed and configured, add the `Laravel\Scout\Searchable` trait to your Eloquent models to make them searchable. 🎉
+
+This trait will use Laravel’s model observers to keep the data in your model in sync with Meilisearch.
```

Comment:
Call me heartless. I removed your emoji.

More seriously, I think these two work better as a single paragraph so it reinforces the connection between "this trait" and the trait you're referring to.

```suggestion
With Scout installed and configured, add the `Laravel\Scout\Searchable` trait to your Eloquent models to make them searchable. This trait will use Laravel’s model observers to keep the data in your model in sync with Meilisearch.
```